deepchem/models/tensorgraph/layers.py

```diff
@@ -3066,7 +3120,10 @@ def AlphaShare(in_layers=None, **kwargs):
   output_layers = []
   alpha_share = AlphaShareLayer(in_layers=in_layers, **kwargs)
   num_outputs = len(in_layers)
-  return [LayerSplitter(x, in_layers=alpha_share) for x in range(num_outputs)]
+  for num_layer in range(0, num_outputs):
```

Comment:
why?

Reply:
I have no idea why AlphaShare was changed. This is odd, may have been a stale merge.